CONTRIBUTION.md

```diff
@@ -71,6 +71,8 @@ git tools.
   - Format your code and fix all the spelling/grammatical mistakes.
   - Limit the column length of the code and the comments within the code to 120 characters.
   - Error messages within the code should be limited to a single line.
+  - Update the documentation if your PR is introducing or changing user facing functionality
+  - Ensure each PR commit compiles and all unit and e2e tests pass on your local machine 
```

Comment:
Both your statements are missing `.` at the end. Please be consistent with existing documentation.

Reply:
My bad, didn't notice that.